apis/python/src/tiledbsoma/soma_dataframe.py

```diff
@@ -243,11 +241,11 @@ def read_all(
         # TODO: partition,
         # TODO: result_order,
         # TODO: platform_config,
-    ) -> pa.RecordBatch:
+    ) -> pa.Table:
         """
-        This is a convenience method around ``read``. It iterates the return value from ``read`` and returns a concatenation of all the record batches found. Its nominal use is to simply unit-test cases.
+        This is a convenience method around ``read``. It iterates the return value from ``read`` and returns a concatenation of all the table-pieces found. Its nominal use is to simply unit-test cases.
```

Comment:
is this comment still valid?  The PyArrow documentation describes concat_tables, so I'm not sure we need to recapitulate it here.

Or perhaps reword to describe the intent, eg, "Concatenate all partial read results into a single Table"?